src/controllers/idea.controller/deleteIdea/index.js

```diff
@@ -13,19 +13,15 @@ const deleteIdea = catchAsync(async (req, res) => {
     });
   }
 
-  const { id, cognito_sub } = req.body;
-
-  const getUserIdResponse = await Hasura(getUserId, {
-    cognitoSub: cognito_sub,
-  });
+  const { id } = req.body;
 
   const variables = {
     id,
   };
+
   const response = await Hasura(delete_idea, variables);
- 
-  console.log(getUserIdResponse.result.data.user[0].id);
-  insertUserActivity('uploading-idea', 'negative', getUserIdResponse.result.data.user[0].id, [id]);
+
+  insertUserActivity('uploading-idea', 'negative', response.result.data.delete_idea_by_pk.user_id, [id]);
```

Comment:
response.result.data.delete_idea_by_pk.user_id is not returning the user id, check the query in the hasura console also it is retuning null instead of the user id and thats why the activity was not getting inserted in the queue that why added a custom query to get the userId